README.md

```diff
@@ -294,24 +281,49 @@ def mnist_pipeline(
 
 
 pipeline = mnist_pipeline(
-    importer=importer(),
-    trainer=trainer(),
-    evaluator=evaluator(),
+    importer=importer(), # Step 1
+    trainer=trainer(), # Step 2
+    evaluator=evaluator(), # Step 3
 )
+
+# Run the pipeline locally.
 pipeline.run()
 ```
 
-# :racehorse: Get a guided tour with `zenml go`
+This runs the pipeline locally on your machine following the steps defined in the pipeline.
+You can scale this up to run on a full-fledged cloud platform by [switching stacks](https://docs.zenml.io/getting-started/core-concepts) with minimal (or no) code changes!
+
+# 🏇 Get a guided tour with `zenml go`
 
-For a slightly more in-depth introduction to ZenML, taught through Jupyter
+For a more in-depth introduction to ZenML, taught through Jupyter
 notebooks, install `zenml` via pip as described above and type:
 
 ```shell
 zenml go
 ```
 
-This will spin up a Jupyter notebook that showcases the above example plus more
-on how to use and extend ZenML.
+This will spin up a Jupyter notebook that showcases the above example and how to extend ZenML.
+
+# 🍰 ZenBytes
+
+ZenBytes is a series of short practical MLOps lessons using ZenML and its various integrations. 
+It covers many of the core concepts widely used in ZenML and MLOps in general.
+If you're new to ZenML this is a good place to start.
+
+After understanding the example in [Quickstart](#🚅-quickstart), your next port of call is the [fully-fleshed-out quickstart
+example](https://github.com/zenml-io/zenml/tree/main/examples/quickstart) and
+then check out [the ZenBytes repository](https://github.com/zenml-io/zenbytes)
+and notebooks.
+
+
+# 📜 ZenFiles
+
+ZenFiles are production-grade ML use-cases powered by ZenML. They are fully
+fleshed out, end-to-end, projects that showcase ZenML's capabilities. They can
+also serve as a template from which to start similar projects.
+
+The ZenFiles project is fully maintained and can be viewed as a sister
+repository of ZenML. Check it out [here](https://github.com/zenml-io/zenfiles).
```

Comment:
Would remove that sentence, IMO it's not quite true, ZenFiles is the least-well-maintained code we have I think 😅 

```suggestion
Check out ZenFiles [here](https://github.com/zenml-io/zenfiles)!
```